### PR TITLE
Add special page

### PR DESCRIPTION
**Pull request by itkg-nanne**
[OO-FEATURE] add special page for node, tabulation, fieldset and style for block

https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1980
https://github.com/open-orchestra/open-orchestra-elastica-bundle/pull/60
https://github.com/open-orchestra/open-orchestra-model-interface/pull/251

### CODE REVIEW

File: Backoffice/Form/DataTransformer/BlockToArrayTransformer.php [hunk 1]
```diff
@@ -36,5 +36,6 @@ public function transform($data)
      */
     public function reverseTransform($value)
     {
+        return new BlockInterface();
```
Comment (alavieille) [inline Backoffice/Form/DataTransformer/BlockToArrayTransformer.php]: ??

File: Backoffice/Form/Type/BlockType.php [hunk 2]
```diff
@@ -90,4 +154,21 @@ public function getName()
         return 'oo_block';
     }
 
+    /**
+     *
```
Comment (alavieille) [inline Backoffice/Form/Type/BlockType.php]: remove this line

File: Backoffice/Form/Type/NodeType.php [hunk 3]
```diff
@@ -64,8 +68,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'attr' => array(
                     'class' => 'generate-id-source',
                 )
+            ))
+            ->add('specialPageName', 'choice', array(
```
Comment (alavieille) [inline Backoffice/Form/Type/NodeType.php]: required false ?

**Comment by itkg-nanne** [comment]
update done @alavieille

**Comment by alavieille** [comment]
:+1:

**Pull request merged**